modules/back-end/src/Domain/SemanticPatch/FlagComparer.cs

```diff
@@ -306,13 +326,6 @@ public static IEnumerable<FlagInstruction> CompareRule(TargetRule original, Targ
             instructions.Add(new RuleNameInstruction(value));
         }
 
-        // compare dispatch key
-        if (original.DispatchKey != current.DispatchKey)
-        {
-            var value = new RuleDispatchKey { RuleId = ruleId, DispatchKey = current.DispatchKey };
-            instructions.Add(new RuleDispatchKeyInstruction(value));
-        }
-
```

Comment:
Any reason to move the dispatch key comparison down?

Reply:
We usually say: "serve true: 70%, false: 30% Dispatch by xxx"
![image](https://github.com/featbit/featbit/assets/88151306/53ac7d8a-871e-4558-81d5-d11b13d55d05)